packages/api-reference/src/components/Content/ClientLibraries/useFeatured.ts

```diff
@@ -0,0 +1,46 @@
+import { useHttpClientStore, type HttpClientState } from '@/stores'
+import type { ClientId, Target } from '@scalar/snippetz'
+
+export type FeaturedClient = { targetKey: Target['key']; clientKey: ClientId<Target['key']> }
+
+export function useFeatured() {
```

Comment:
nice

can we call this `useFeaturedHttpClients` as a more descriptive name?
and add a comment describing what it does?
and add one or two tests?

👀

Reply:
Yeah sure, I was just consolidating the logic while doing the a11y stuff